packages/react-experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -347,11 +358,14 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
           input.current.inputElement === document.activeElement &&
           (input.current as Autofill).cursorLocation === 0
         ) {
+          const item = selectedItems[selectedItems.length - 1];
+          onAnnounceDeletedRecipients(item.text);
```

Comment:
>text [](http://example.com/codeflow?start=43&length=4)

Build break, .text is not guaranteed to be on `T`.

Unsure how to address it, so posting a PR so we can more easily see the change to address the build break